internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -427,8 +431,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, err
 	}
 
-	origRV := ""
-	if err := r.client.Apply(ctx, crd, resource.MustBeControllableBy(d.GetUID()), resource.StoreCurrentRV(&origRV)); err != nil {
+	// We are aware that using controller-runtime server-side apply will result in
+	// some zero-value fields, however, this should not affect the way CRDs are
+	// implemented in this controller. For a discussion on the runtime issue, see
+	// https://github.com/kubernetes-sigs/controller-runtime/issues/347.
```

Comment:
@turkenh Could you do a quick sanity check on this. I know you went pretty deep on SSA issues recently for provider-kubernetes.

I feel okay switching to SSA here as long as we don't see any issues applying CRDs in practice (which appears to be the case in @dalton-hill-0's testing.)

Reply:
Just a friendly reminder 🙂
@turkenh

Reply:
@dalton-hill-0 It looks like @turkenh is on vacation. He'll be back in a week.

Reply:
This sounds good to me. 

To see whether SSA causing any immediate problem, a quick sanity test could be leaving a watch request for a CRD of an XRD open for a while, e.g. `kubectl get crds <crd-name> -w`, and checking whether it gets periodic updates or not (i.e. resourceVersion keeps increasing). You may restart Crossplane or add some labels to XRDs to trigger new reconciliations.

Reply:
I performed the sanity checks above and all behavior was expected.
- no increase to resource version on crossplane restart
- no increase to resource version when adding annotations to xrds (they do not get copied over from meta)
- resource version increased when adding label to xrd (copies over to crds)
- no infinite patch loop between crossplane and cert manager